erpnext/accounts/doctype/subscription/subscription.py

```diff
@@ -148,8 +148,7 @@ def get_billing_cycle_and_interval(self):
 		billing_info = frappe.db.sql(
 			"select distinct `billing_interval`, `billing_interval_count` "
 			"from `tabSubscription Plan` "
-			"where name in %s",
-			(plan_names,),
+			"where name in {plan_names}".format(plan_names="('" + "','".join(plan_names) + "')"),
```

Comment:
Is this a problem with postgres `IN` queries? if so we should handle it in framework and not here

Reply:
created frappe issue: https://github.com/frappe/frappe/issues/17231

Reply:
Fixed by rewriting query